cmd/mesh/manifest-versions.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	versionsMapURL = "https://raw.githubusercontent.com/istio/operator/master/version/versions.yaml"
+	versionsMapURL = "https://raw.githubusercontent.com/istio/operator/release-1.3/version/versions.yaml"
```

Comment:
I'm not sure whether this change is required or correct.

At least, I'm sure updating the tags `master-latest-daily` -> `release-1.3-latest-daily` is required.

Reply:
Thanks for doing this. In this case it is not required because master always contains the version of the map we want users to have. That's because they may want to know about release versions more recent than their binary.

Reply:
Ok, I'll revert that line.